quizcon/templates/main/lti_assignment.html

```diff
@@ -4,49 +4,120 @@
 {% block title %}{{quiz.title|truncatewords:3}}{% endblock %}
 
 {% load coursetags static %}
-
+{% block topnavbar %}
+<nav class="navbar navbar-expand-lg navbar-dark">
+    <div class="container-fluid">
+        <a class="navbar-brand" href="/course/1" title="Home"><strong><span class="text-light">QuizCon</span></strong> </a>
```

Comment:
Hmmm....our LTI views should not link back into the main QuizCon application for students, but it might be super helpful for faculty to have an Edit link or Dashboard link?

If we continue to have these links for instructors, they definitely need to open a new tab as our main views are not configured to work in an iFrame. The links should be constructed like so:

`<a href="#" target="_blank" rel="noopener noreferrer">Link</a>`